dbmongo/lib/bdf/main.go

```diff
@@ -84,6 +92,21 @@ func Parser(cache marshal.Cache, batch *base.AdminBatch) (chan marshal.Tuple, ch
 				}
 				bdf := BDF{}
 				bdf.Siren = strings.Replace(row[0], " ", "", -1)
+
+				validSiren := sfregexp.RegexpDict["siren"].MatchString(bdf.Siren)
+				if !validSiren {
+					tracker.Error(errors.New("siren invalide : " + bdf.Siren))
+					tracker.Next()
+					continue
+				}
+
+				filtered, err := marshal.IsFiltered(bdf.Siren, filter)
+				tracker.Error(err)
+				if filtered {
+					tracker.Next()
```

Comment:
Même si on l'a pas fait systématiquement dans les autres parseurs, ne devrait-on pas aussi créer une FilterError dans ce cas ?

```suggestion
					tracker.Error(base.NewFilterError(err))
					tracker.Next()
```